### PR TITLE
[docs]: Guide on working with Etsy in Gatsby

### PR DESCRIPTION
**Pull request by bball07**
## Description

Reference Guide on working with Etsy in Gatsby.

## Related Issues

New guide on working with Etsy #20226

### CODE REVIEW

File: www/src/data/sidebars/doc-links.yaml [hunk 1]
```diff
@@ -469,6 +469,8 @@
                   link: /docs/processing-payments-with-stripe/
                 - title: Processing Payments with Square
                   link: /docs/processing-payments-with-square/
+                - title: Guide on working with Etsy in Gatsby
```
Comment (laurieontech) [inline www/src/data/sidebars/doc-links.yaml]:
```suggestion
                - title: Working with Etsy in Gatsby
```

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+Title: Guide on working with Etsy in Gatsby
```
Comment (laurieontech) [inline docs/docs/working-with-Etsy-in-Gatsby.md]:
```suggestion
Title: Working with Etsy in Gatsby
```
The title should be able to focus on what technologies are in use without worrying about what kind of document it is.

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+# Guide on working with Etsy in Gatsby
```
Comment (laurieontech) [inline docs/docs/working-with-Etsy-in-Gatsby.md]:
```suggestion
```
No need to duplicate the title in the main content. Also, the title in the frontmatter is automatically an h1, so for accessibility purposes we would not want to have another h1 in this document.

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+## Etsy in Gatsby
+
+This guide will walk you through the steps needed in order to successfully work with Etsy e-commerce sites in Gatsby.
```
Comment (laurieontech) [inline docs/docs/working-with-Etsy-in-Gatsby.md]: "steps needed" sounds more like a tutorial than a reference guide. Take a look at the [reference guide templates](https://www.gatsbyjs.org/contributing/docs-templates/#reference-guides) and make sure this document is aligned with those recommendations.

Comment (marcysutton) [reply docs/docs/working-with-Etsy-in-Gatsby.md]: This was marked as resolved, but I don't see changes yet–@bball07 are you updating the PR to work more as a Reference Guide?

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+## Prerequisites
+
+- Create a new [Etsy account](https://www.etsy.com/join) and store if you don’t have one.
+- Create a private app in your store by navigating to Apps, then Manage private apps.
+- Create a new private app, with any “Private app name” and leaving the default permissions as read access under Admin API.
+- Enable the Etsy API by checking the box that says “Allow this app to access your storefront data using Storefront API”.
+- Make sure to also grant access to read product and customer tags by checking their corresponding boxes.
+
+## If you do not already have one ready, [create a Gatsby site](/docs/quick-start):
```
Comment (AishaBlake) [inline docs/docs/working-with-Etsy-in-Gatsby.md]: Creating a Gatsby site probably doesn't need to be linked _and_ explained in quite so much detail. It also probably doesn't need to be an `h2`. If you mention it at all, I'd shrink this whole section down to a link.

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
+---
+Title: Working with Etsy in Gatsby
+---
+
+## Etsy in Gatsby
+
+This guide will walk you through the steps needed in order to successfully work with Etsy e-commerce sites in Gatsby.
+
+## Prerequisites
```
Comment (AishaBlake) [inline docs/docs/working-with-Etsy-in-Gatsby.md]: This is in another comment but I'd try to move away from the step-by-step bullets where it makes sense.

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+- Create a new [Etsy account](https://www.etsy.com/join) and store if you don’t have one.
+- Create a private app in your store by navigating to Apps, then Manage private apps.
```
Comment (AishaBlake) [inline docs/docs/working-with-Etsy-in-Gatsby.md]: I wasn't able to find this in the UI. I would direct people to "[Register as a developer](https://www.etsy.com/developers/register)".

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+- Create a new private app, with any “Private app name” and leaving the default permissions as read access under Admin API.
+- Enable the Etsy API by checking the box that says “Allow this app to access your storefront data using Storefront API”.
+- Make sure to also grant access to read product and customer tags by checking their corresponding boxes.
+
```
Comment (AishaBlake) [inline docs/docs/working-with-Etsy-in-Gatsby.md]: You also need to enable two-factor authentication in order to create an API key.

**Comment by AishaBlake** [comment]
Sent this without a comment unintentionally. Roll with these suggestions for and maybe Marcy can help if there's no option to open a test store without providing your bank info. We should make sure each section is as accurate as possible before merging!

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+        language: "en", // optional
+      },
+    },
+  ],
+}
+```
+
+## Etsy API
+
+Etsy has a public API that developers can [query](/packages/gatsby-source-etsy/#example-graphql-queries) and use to update specific information from Etsy. The Etsy API provides a simple [RESTful](https://en.wikipedia.org/wiki/Representational_State_Transfer) interface with lightweight [JSON](https://en.wikipedia.org/wiki/JSON)-formatted responses to use many of Etsy's website features, including public profiles, shops, listings, tags, favorites, and sales data, using [OAuth](https://www.etsy.com/developers/documentation/getting_started/oauth) to allow both read and write access to users' public and private data. If you're a developer and want to access the Etsy API, start by [creating an Etsy account](https://help.etsy.com/hc/articles/115015568007) if you don't have one already. Once registered, you can obtain a key at the [Etsy Developer Community website](https://www.etsy.com/developers?segment=selling). After you receive an API key, we’d recommend reviewing the Etsy [documentation](https://www.etsy.com/developers/documentation) to get started.
```
[the remark this answers is not in the document]
Comment (marcysutton) [reply docs/docs/working-with-Etsy-in-Gatsby.md]: Some of this boilerplate info isn't relevant to Gatsby sites, either, so I'd limit to what is critical to creating a Gatsby site with Etsy.

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
+---
+Title: Working with Etsy in Gatsby
+---
+
+## Etsy in Gatsby
```
Comment (marcysutton) [inline docs/docs/working-with-Etsy-in-Gatsby.md]: I don't think you need this heading, as it's redundant to the page title.

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+
+- From your terminal, you will want to cd or change directory into the folder where you would want your project to live.
+- In your terminal put in the command `npm install -g gatsby -cli` to install the gatsby command-line interface.
+- Next, you would want to create a new project using the command gatsby new [the name of your site].
+- You would then want to cd into your project folder using cd [the name of your site].
+- Run `gatsby develop` to view your project locally.
+
+## [Gatsby-source-etsy ](/packages/gatsby-source-etsy/)
+
+While in your terminal type the command `npm i gatsby-source-etsy`. You will then need to add the plugin to your gatsby-config.js file:
```
Comment (marcysutton) [inline docs/docs/working-with-Etsy-in-Gatsby.md]: You should mention the `apiKey` and `shopId` in context here. The Shopify guide walks the user through installing its relevant plugin and mentions filling these in: https://www.gatsbyjs.org/docs/building-an-ecommerce-site-with-shopify/

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+        shopId: "your shop id here",
+
+        language: "en", // optional
+      },
+    },
+  ],
+}
+```
+
+## Etsy API
```
Comment (marcysutton) [inline docs/docs/working-with-Etsy-in-Gatsby.md]: I'd expect a Gatsby/Etsy guide would show the user how to pull Etsy listing data into a component, at the very least. Can you add an example showing how this works? The README has some details, they could inspire the guide but should include content written specifically for this learning resource (reference material on how to set up a Gatsby site with an Etsy shop) https://www.gatsbyjs.org/packages/gatsby-source-etsy/

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+
+## Opportunity for a new plugin/starter
+
+Want to create a new Etsy starter or plugin for Gatsby? Check out our [contributing guide](/contributing/)!
+
+## Other resources
+
+[Gatsby Shopify Starter](/starters/AlexanderProd/gatsby-shopify-starter/)
```
Comment (marcysutton) [inline docs/docs/working-with-Etsy-in-Gatsby.md]: This starter seems irrelevant to an Etsy guide.

File: www/src/data/sidebars/doc-links.yaml (hunk 1 above)
Comment (marcysutton) [reply www/src/data/sidebars/doc-links.yaml]: I am marking this as unresolved, as it needs to be adjusted as part of this PR.

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+      price
+
+      url
+    }
+  }
+}
+```
+
+Take a look at the [API Documentation](https://www.etsy.com/developers/documentation/reference/listing) for more information on listing data in Etsy.
+
```
[the remark this answers is not in the document]
Comment (marcysutton) [reply docs/docs/working-with-Etsy-in-Gatsby.md]: I just thought of something! The Additional Resources section should link to the Etsy example so users have something to kickstart their work. The repo you created needs be finished so it can accompany this documentation. The README and package.json files also needed updating for the example last I checked. https://github.com/bball07/gatsby-etsy-site

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 15]
```diff
@@ -0,0 +1,68 @@
+---
+Title: Working with Etsy in Gatsby
```
Comment (AishaBlake) [inline docs/docs/working-with-Etsy-in-Gatsby.md]:
```suggestion
title: Working with Etsy in Gatsby
```

The function that generates these pages is looking for "title". This change gets the title to show up at the top of the guide.

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+This guide explains how to begin working with the e-commerce site Etsy in Gatsby.
+
+## Etsy API
+
+Utilizing the Etsy API (Application Programming Interface), developers and non-developers can create apps for the web that pull in information such as BillCharge, CartListing, ShippingOption, and a list of other API references available to query. First, you must create a [developer account](<[https://www.etsy.com/developers/register](https://www.etsy.com/developers/register)>) with Etsy, confirm the account, and enable two-factor authentication. This will allow you to generate and receive your own API key. You will be asked for a short description of your app.
```
[the remark this answers is not in the document]
Comment (AishaBlake) [reply docs/docs/working-with-Etsy-in-Gatsby.md]:
```suggestion
Utilizing the Etsy API (Application Programming Interface), developers and non-developers can create apps for the web that pull in information such as BillCharge, CartListing, ShippingOption, and a list of other API references available to query. First, you must create a [developer account](https://www.etsy.com/developers/register) with Etsy, confirm the account, and enable two-factor authentication. This will allow you to generate and receive your own API key. You will be asked for a short description of your app.
```

They get interpreted as relative links, which leads to a 404.

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+
+After you receive an API key, it’s recommended to review the Etsy [documentation](<[https://www.etsy.com/developers/documentation](https://www.etsy.com/developers/documentation)>) on how to get started. Most importantly, in order for your app to function, you will need your Etsy [ShopId](<[https://support.cartrover.com/portal/kb/articles/how-to-get-your-etsy-shop-id](https://support.cartrover.com/portal/kb/articles/how-to-get-your-etsy-shop-id)>) which is the unique number associated with your store.
+
+## If you do not already have one ready, create a Gatsby site:
+
+In your terminal, type the command ‘npm install -g gatsby -cli’ to install the gatsby command-line interface. Create a new project using the command ‘gatsby new [the name of your site]’. Run ‘gatsby develop’ to view your project locally.
+
+## [Gatsby-source-etsy ](https://www.gatsbyjs.org/packages/gatsby-source-etsy/)
+
+While in your terminal, type the command ‘npm i gatsby-source-etsy’. You will then need to add the plugin to your ‘gatsby-config.js’ file, your API key, and Shop ID:
```
[the remark this answers is not in the document]
Comment (AishaBlake) [reply docs/docs/working-with-Etsy-in-Gatsby.md]: I made a similar note earlier but I think you want to combine what we've each said like this:

While in your terminal, type the command:

```bash
npm install gatsby-source-etsy
```

You will then need to add the plugin to your `gatsby-config.js` file along with your API key and Shop ID:

... When you _do_ use single back-ticks, make sure they're actual back-ticks. It looks like there may still be some formatting issues that got carried over from Google Docs.

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+}
+```
+
+Pull your Etsy listing data into a component by creating a GraphQL query and specify what data you would like to be generated by calling out the specific field you would like to display.
+
+```graphql
+{
+  allFeaturedEtsyListing(
+    sort: { fields: featured_rank, order: ASC }
+
```
[the remark this answers is not in the document]
Comment (AishaBlake) [reply docs/docs/working-with-Etsy-in-Gatsby.md]: These don't actually come through when the page is rendered. Something to keep in mind if you _are_ trying to emphasize anything in particular.

File: www/src/data/sidebars/doc-links.yaml [hunk 19]
```diff
@@ -469,6 +469,8 @@
                   link: /docs/processing-payments-with-stripe/
                 - title: Processing Payments with Square
                   link: /docs/processing-payments-with-square/
+                - title: Working with Etsy in Gatsby
+                  link: /docs/working-with-Etsy-in-Gatsby/
```
Comment (AishaBlake) [inline www/src/data/sidebars/doc-links.yaml]: This isn't actually hurting anything and may only be personal preference but I'd keep filenames lowercase for consistency. That would mean a change here and renaming the Markdown file itself.

```suggestion
                  link: /docs/working-with-etsy-in-gatsby/
```

File: docs/docs/working-with-Etsy-in-Gatsby.md [hunk 20]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+This guide explains how to begin working with the e-commerce site Etsy in Gatsby.
+
+## Etsy API
+
+Utilizing the Etsy API (Application Programming Interface), developers and non-developers can create apps for the web that pull in information such as BillCharge, CartListing, ShippingOption, and a list of other API references available to query. First, you must create a [developer account](<[https://www.etsy.com/developers/register](https://www.etsy.com/developers/register)>) with Etsy, confirm the account, and enable two-factor authentication. This will allow you to generate and receive your own API key. You will be asked for a short description of your app.
+
+After you receive an API key, it’s recommended to review the Etsy [documentation](<[https://www.etsy.com/developers/documentation](https://www.etsy.com/developers/documentation)>) on how to get started. Most importantly, in order for your app to function, you will need your Etsy [ShopId](<[https://support.cartrover.com/portal/kb/articles/how-to-get-your-etsy-shop-id](https://support.cartrover.com/portal/kb/articles/how-to-get-your-etsy-shop-id)>) which is the unique number associated with your store.
```
[the remark this answers is not in the document]
Comment (AishaBlake) [reply docs/docs/working-with-Etsy-in-Gatsby.md]: In addition, you might want to lay out some of the issues and questions that came up for you. For example, the process of figuring out whether you need full access or if you should open your shop in development mode might be useful information. There's no explicit mention of how or when to start a shop. Those sorts of things might fit well in a "Prerequisites" section.

File: docs/docs/working-with-Etsy-in-Gatsby.md (hunk 15 above)
Comment (marcysutton) [reply docs/docs/working-with-Etsy-in-Gatsby.md]: Good find, @AishaBlake. That’s another case where it should be discovered by running the docs site locally; same  with the other formatting issues.

File: docs/docs/working-with-Etsy-in-Gatsby.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (bball07) [reply docs/docs/working-with-Etsy-in-Gatsby.md]: The Additional Resources section with the example will get added after my meeting tomorrow with Sid.

File: docs/docs/sourcing-from-etsy.md [hunk 21]
```diff
@@ -0,0 +1,113 @@
...
+
+[Etsy](https://www.etsy.com/) is an online marketplace for buying and selling handmade, vintage, and custom items. It's long been a home for creative entrepreneurs and can be used as a data source for your Gatsby site.
+
+## Prerequisites
+
+This guide assumes you already have a functioning Gatsby site as well as an open Etsy shop. You can start a new Gatsby site in a few steps with the [Quick Start guide](/docs/quick-start) and then [start selling on Etsy](https://www.etsy.com/your/shop/create)!
+
+## Getting an API key
+
+You'll need to make sure you have two-factor authentication enabled for your Etsy account. If you don't already, you'll be prompted when you go to [register a new application](https://www.etsy.com/developers/register). Fill in the required information as completely as you can, read Etsy's terms and conditions, and create your app.
```
Comment (laurieontech) [inline docs/docs/sourcing-from-etsy.md]:
```suggestion
You'll need to make sure you have two-factor authentication enabled for your Etsy account. If you don't already have it, you'll be prompted when you go to [register a new application](https://www.etsy.com/developers/register). Fill in the required information as completely as you can, read Etsy's terms and conditions, and create your app.
```

File: docs/docs/sourcing-from-etsy.md [hunk 22]
```diff
@@ -0,0 +1,113 @@
...
+        }
+      }
+    }
+  }
+}
+```
+
+Using this query on a page or in a component will allow you to display your listings. The following example loops through the featured items and displays them within a component.
+
+```js
```
Comment (laurieontech) [inline docs/docs/sourcing-from-etsy.md]:
```suggestion
```jsx
```

File: docs/docs/sourcing-from-etsy.md [hunk 23]
```diff
@@ -0,0 +1,113 @@
...
+      <a href={item.url}>
+        <h2>{item.title}</h2>
+      </a>
+      <Img fixed={item.childEtsyListingImage.childFile.childImageSharp.fixed} />
+      <p>${item.price}</p>
+      <p>{item.description}</p>
+    </>
+  ))
+}
+```
```
Comment (laurieontech) [inline docs/docs/sourcing-from-etsy.md]: Should we put this example in the context of a component at all?

File: docs/docs/sourcing-from-etsy.md [hunk 24]
```diff
@@ -0,0 +1,113 @@
...
+Using this query on a page or in a component will allow you to display your listings. The following example loops through the featured items and displays them within a component.
+
+```js
+{
+  data.allFeaturedEtsyListing.nodes.map(item => (
+    <>
+      <a href={item.url}>
+        <h2>{item.title}</h2>
+      </a>
+      <Img fixed={item.childEtsyListingImage.childFile.childImageSharp.fixed} />
```
Comment (laurieontech) [inline docs/docs/sourcing-from-etsy.md]: Is there a way to add variable alt text here?

File: docs/docs/sourcing-from-etsy.md [hunk 25]
```diff
@@ -0,0 +1,113 @@
...
+    }
+  }
+}
+```
+
+Using this query on a page or in a component will allow you to display your listings. The following example loops through the featured items and displays them within a component.
+
+```js
+{
+  data.allFeaturedEtsyListing.nodes.map(item => (
```
Comment (laurieontech) [inline docs/docs/sourcing-from-etsy.md]: I believe this will throw a warning in React because there isn't a unique key for each item. We may not care since it's a guide, but perhaps an index as key makes sense to include?

File: docs/docs/sourcing-from-etsy.md [hunk 26]
```diff
@@ -0,0 +1,113 @@
...
+      <p>${item.price}</p>
+      <p>{item.description}</p>
+    </>
+  ))
+}
+```
+
+## Gatsby advantages
+
+If you are familiar with Etsy, you are aware of the restrictions around branding within the platform. Shops have an extremely similar look, which limits the creativity any individual shop owner can display. In addition, you have no control over Etsy's policies. An increase in their listing fees could have a significant impact on your profits. If Etsy went out of business, you would quickly be left without any customers.
```
Comment (laurieontech) [inline docs/docs/sourcing-from-etsy.md]:
```suggestion
If you are familiar with Etsy, you are aware of the restrictions around branding within the platform. Shops have an extremely similar look, which limits the creativity any individual shop owner can display. In addition, you have no control over Etsy's policies.
```
Might remove the last two sentences. I think we can highlight the advantages without detailing future concerns with Etsy.

File: docs/docs/sourcing-from-etsy.md [hunk 27]
```diff
@@ -0,0 +1,113 @@
...
+
+So why sell on Etsy at all? The answer is that Etsy shops trade individuality for discoverability. Etsy's advertising budget is enormous compared to what a shop owner, operating alone, could afford. You also benefit from the trust Etsy has built up over the years. Because people are familiar with Etsy, they'll often go straight there to purchase their handmade or vintage items.
```
Comment (laurieontech) [inline docs/docs/sourcing-from-etsy.md]:
```suggestion
However, there are great benefits to selling on Etsy. What you lose in individuality you gain in discoverability. Etsy's advertising budget is enormous compared to what a shop owner, operating alone, could afford. You benefit from the trust Etsy has built up over the years. Because people are familiar with Etsy, they'll often go straight there to purchase their handmade or vintage items.
```

File: docs/docs/sourcing-from-etsy.md (hunk 23 above)
Comment (AishaBlake) [reply docs/docs/sourcing-from-etsy.md]: I didn't, similar to the "Sourcing from WooCommerce" one, because it could go wherever. It might make sense in an `index.js`..?

Comment (laurieontech) [reply docs/docs/sourcing-from-etsy.md]: I get the lack of file/component specified, I was thinking more a minimal wrapper ```const SomePage = ({ data }) => {``` kind of thing. But not strictly necessary.

File: docs/docs/sourcing-from-etsy.md (hunk 24 above)
Comment (AishaBlake) [reply docs/docs/sourcing-from-etsy.md]: So there's not a _great_ solution for this. Etsy actually doesn't give shop owners a place to add alt text. (Another reason to pull your listings into your own site!) The best I could come up with to do this with the data provided is repeating the title of the item.

I was thinking the alt text was taken from the filename by default but I may have been thinking of MDX... As it is, the alt attribute is still added but it's empty, which means the images should be ignored. Given that the image is sandwiched between the item's name and description, I'm not sure if it makes sense to repeat that information or leave it as is. Tagging @madalynrose!

Comment (AishaBlake) [reply docs/docs/sourcing-from-etsy.md]: Either way, I think it's worth noting in the text.

Comment (madalynrose) [reply docs/docs/sourcing-from-etsy.md]: +1 what Aisha said. if the alt text is going to be repeating information already present, that can get really annoying with a screen reader. keeping the image purely presentational makes sense unless we can communicate extra information about it (e.g. color variation)

Comment (laurieontech) [reply docs/docs/sourcing-from-etsy.md]: Makes sense! Hadn't thought about the images being sandwiched between that info. Thanks for thinking it through so thoroughly :)

File: docs/docs/sourcing-from-etsy.md [hunk 28]
```diff
@@ -0,0 +1,116 @@
...
+      </a>
+      <Img fixed={item.childEtsyListingImage.childFile.childImageSharp.fixed} />
+      <p>${item.price}</p>
+      <p>{item.description}</p>
+    </section>
+  ))
+}
+```
+
+Note that, in this example, there is no alt text passed to the `Img` component because this image can be considered presentational. Etsy doesn't provide space for you specify [text alternatives](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#text-alternatives) for your images. Instead, when you list an item on Etsy, alt text is generated for each image you associate with that item from its title. Including the title or description as alt text here would only repeat the exact same information for anyone using a screen reader.
```
Comment (laurieontech) [inline docs/docs/sourcing-from-etsy.md]:
```suggestion
Note that, in this example, there is no alt text passed to the `Img` component because this image can be considered presentational. All images appear in association with their title and description. Therefore, including the same information as alt text would be repetitive for anyone using a screen reader.
```
I had a slight rewording for this and then I wondered whether it makes more sense to ignore why Etsy doesn't do this and only mention the example above? So that's the recommendation I made, but open to alternatives.

**Comment by laurieontech** [comment]
One last thing based on the alt text explanation. But other than that this is good to go!

**Comment by laurieontech** [comment]
Looks great!
